components/07-layout/layout.hbs

```diff
@@ -1,5 +1,5 @@
 <h1>The Hague (h1)</h1>
-<p>The Hague (<a href="https://en.wikipedia.org/wiki/Dutch_language" class="extern" target="_blank">Dutch</a>: Den Haag) is a city on the western coast of the Netherlands and the capital of the province of South Holland.</p>
+<p>The Hague (<a href="https://en.wikipedia.org/wiki/Dutch_language" class="extern" target="_blank">Dutch</a>: Den Haag) is a city on the western coast of the Netherlands and the <a href="#" class="download">capital</a> of the province of South Holland.</p>
```

Comment:
class="download"? dat lijkt me sterk

Reply:
Waarom?

Reply:
Omdat ik bij het woord "capital" niet snel een download verwacht. Een relevanter voorbeeld lijkt me iets als "Our findings have been summarized in this report", waarbij "this report" dan een download is.

Reply:
Nou vooruit dan maar